pkg/defaults/policies/files/cvss_6_privileged.json

```diff
@@ -1,7 +1,7 @@
 {
   "id": "93f4b2dd-ef5a-419e-8371-38aed480fb36",
-  "name": "Fixable CVSS >= 6 and Privileged",
-  "description": "Alert on deployments running in privileged mode with fixable vulnerabilities with a CVSS of at least 6",
+  "name": "Privileged containers with fixable CVEs and CVSS >= 6",
```

Comment:
alerts for policy violations are raised on deployment level. hence should be worded accurately - Deployment with Privileged containers with fixable CVEs and CVSS >= 6